provider/aws/formation/rack.json

```
@@ -2590,10 +2721,19 @@
               { "Name": "RACK", "Value": { "Ref": "AWS::StackName" } },
               { "Name": "STACK_ID", "Value": { "Ref": "AWS::StackId" } }
             ],
-            "Image": { "Fn::If": [ "BlankBuildImage",
-              { "Fn::Sub": "convox/rack:${Version}" },
-              { "Ref": "BuildImage" }
-            ] },
+            "Image": {
+              "Fn::If": [
+                "BlankBuildImage",
+                {
+                  "Fn::If": [
+                    "InstanceARM",
+                    { "Fn::Sub": "convox:rack:${Version}-arm64" },
```

Typo on the image definition:

```suggestion
                    { "Fn::Sub": "convox/rack:${Version}-arm64" },
```